src/scripts/OSFramework/OSUI/Pattern/Dropdown/ServerSide/DropdownServerSide.ts

```diff
@@ -451,6 +446,26 @@ namespace OSFramework.OSUI.Patterns.Dropdown.ServerSide {
 			}
 		}
 
+		// Method that will check if the BalloonOptionsWrapper should be moved to outside of the pattern context
+		private _shouldBalloonOptionsBeMoved(): boolean {
+			/* NOTE:
+				- When inside BottomSheet the BalloonOptionsWrapper should be moved to the content wrapper
+				due to position issues related with fixed position of the balloon against BottomSheet fixed position.
+				More info at Release Note: ROU-11549
+			 */
+			// Ensure DropdownServerSide is inside at BottomSheet, Notification or Sidebar
+			if (
```

Comment:
@joselrio Why not simply do the following:
`return (
        Helper.DeviceInfo.IsPhone &&
        (this.selfElement.closest(Enum.InsidePattern.BottomSheet) ||
         this.selfElement.closest(Enum.InsidePattern.Notification) ||
         this.selfElement.closest(Enum.InsidePattern.Sidebar))
    );`

Reply:
Just code readability...